deeprankcore/query.py

```diff
@@ -508,19 +473,16 @@ def __hash__(self) -> hash:
 
     @staticmethod
     def _get_atom_node_key(atom) -> str:
-        """Pickle has problems serializing the graph when the nodes are atoms,
-        so use this function to generate an unique key for the atom"""
+        """Pickle has problems serializing the graph when the nodes are atoms,so use this function to generate an unique key for the atom"""
```

Comment:
```suggestion
        """Since pickle has problems serializing the graph when the nodes are atoms, this function can be used to generate a unique key for the atom."""
```